app/src/main/java/com/google/droidjump/models/LevelManager.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump.models;
+
+import static com.google.droidjump.GameConstants.FIRST_LEVEL_ID;
+import static com.google.droidjump.GameConstants.GAME_LEVELS_COUNT;
+import static com.google.droidjump.GameConstants.GAME_VIEW_CURRENT_LEVEL_STRING;
+import static com.google.droidjump.GameConstants.GAME_VIEW_DATA;
+import static com.google.droidjump.GameConstants.GAME_VIEW_LAST_LEVEL_STRING;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+public class LevelManager {
+    private static int levelsCount;
```

Comment:
this is kind of fixed. can we just use GAME_LEVELS_COUNT?

Reply:
@redls GAME_LEVELS_COUNT is only a temporary solution. If I understand correctly, count of levels will be extracted from JSON configuration later. Is it right @Daniel-ND ?

Reply:
Yes, levelsCount will be based on data stored in config file

Reply:
Added todo method for this purpose.

Reply:
oh I see.